containers_test.go

```diff
@@ -0,0 +1,23 @@
+package proxmox
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStartContainer(t *testing.T) {
+	client := ClientFromLogins()
+	node, err := client.Node("")
+	assert.Nil(t, err)
+
+	container, err := node.Container(103)
```

Comment:
so after getting a node and starting an arbitrary container let's boot nginx or something and loop waiting for the welcome to nginx screen or something.

something like this locally will show the `Welcome to nginx!` locally.. probably can convert to an lxc call and hit the node ip and port and wait for a 200OK + `Welcome to nginx!`

```
docker run -d --name myNGINX_1 -p 8081:80 --restart=always nginx
```